### PR TITLE
[DOCS-6035] UTM campaign guide for RUM

### PR DESCRIPTION
**Pull request by jamiemilstein**
Guide for RUM on how to monitor UTM campaigns

### CODE REVIEW

**Comment by buraizu** [comment]
Thanks, created DOCS-6035 to review

File: content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+description: Learn how to use RUM & Session Replay to monitor the performance of UTM campaigns.
+---
+
+## Overview
+
+Urchin Tracking Module (UTM) tracking is a parameter that can be added to a URL for tracking the performance of specific campaigns and identifying attribution paths for how visitors arrived on your website. This guide walks you through the types of UTM parameters Datadog RUM collects and how you can use RUM to monitor their use.
+
+## Data collected
+
+UTM campaigns are connected to [View][1] events in RUM. These are collected automatically by the Browser SDK and can be viewed as facets in the RUM explorer. The UTM parameters Datadog collects can be defined as the following:
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum:13]: @jamiemilstein In the second sentence, can we clarify what "These" refers to? Is it the UTM campaigns or the View events?

Comment (rtrieu) [inline content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum:13]:
```suggestion
UTM campaigns are connected to [View][1] events in RUM. These are collected automatically by the Browser SDK and can be viewed as facets in the RUM Explorer. The UTM parameters Datadog collects can be defined as the following:
```

File: content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+| Field                | Type   | Description                                                   |
+|-------------------------------|--------|---------------------------------------------------------------|
+| `view.url_query.utm_source`     | string | The parameter in the URL tracking the source of traffic. |
+| `view.url_query.utm_medium`        | string | The parameter in the URL tracking the channel where the traffic is coming from.    |
+| `view.url_query.utm_campaign`  | string | The paramter in the URL identifying the specific marketing campaign tied to that view.              |
+| `view.url_query.utm_content`  | string | The paramter in the URL identifying the specific element a user clicked within a marketing campaign.           |
```
Comment (rtrieu) [inline content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum:20]:
```suggestion
| `view.url_query.utm_content`  | string | The parameter in the URL identifying the specific element a user clicked within a marketing campaign.           |
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Hi @jamiemilstein , cool guide! I added it to the guide index page and made a few updates. I also left you a few comments. Let me know if you have any questions! Otherwise, it's almost ready to be published!

File: content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum (hunk 1 above)
Comment (jamiemilstein) [reply content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum:13]: would "Campaign data" be clear here?

Comment (rtrieu) [reply content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum]: Yes, will add this.

**Review by rtrieu — APPROVED** [review]
Approving

**Comment by rtrieu** [comment]
@jamiemilstein Is this feature ready and available? Should we merge this?

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**